storage/miner_sealing.go

```diff
@@ -134,7 +134,7 @@ func (m *Miner) SectorsStatus(ctx context.Context, sid abi.SectorNumber, showOnC
 		LastErr: info.LastErr,
 		Log:     log,
 		// on chain info
-		SealProof:          0,
```

Comment:
We really need `SealProof` to be set based on the proof type of the sector from the local store, for when we need information on a sector that is not yet on chain.